test/common_test/joint_features.cc

```diff
@@ -592,13 +590,21 @@ TYPED_TEST(JointFeaturesPositionLimitsForceControlTest, JointSetCombinedLimitsWi
 // TODO(anyone): position limits do not work very well with velocity control
 // bug https://github.com/dartsim/dart/issues/1583
 // resolved in DART 6.11.0
-TYPED_TEST(JointFeaturesPositionLimitsForceControlTest, DISABLED_JointSetPositionLimitsWithVelocityControl)
+TYPED_TEST(JointFeaturesPositionLimitsForceControlTest, JointSetPositionLimitsWithVelocityControl)
 {
   for (const std::string &name : this->pluginNames)
   {
-    if(this->PhysicsEngineName(name) != "dartsim")
+    if (this->PhysicsEngineName(name) == "dartsim")
+    {
+      GTEST_SKIP();
```

Comment:
This is skipping `dartsim` while the original code was skipping all other engines. Is this intentional?

Reply:
yes this is intentional. Previously the test was disabled but I re-enabled it but only for bullet-featherstone